spec/models/user_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user) { User.create(email: 'joebloggs@email.com', name: 'Joe', password: 'password') }
```

Comment:
does a new user get created for every test or does lazy-loading means it only runs once when necessary?